gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,48 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+After installing the xref:policies-pdk-prerequisites.adoc[], you can develop custom policies by using the Policy Development Kit (PDK).
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in PDK. The following docs do not include steps that occur outside of PDK, such as xref:policies-pdk-deploy-policies.adoc[applying a policy].
+
+After you xref:policies-pdk-create-project.adoc[create a new project], you might iteratively complete some steps. For example, while writing the policy's source code, you might need to return to xref:policies-pdk-create-schema-definition.adoc[add additional parameters], or you might need to recompile the policy after debugging the source code.
+
+The commands associated with each step are provided for reference. However, configuration is necessary before running some of the commands.
+
+To create a custom policy with PDK, follow these steps:
+
+. xref:policies-pdk-create-project.adoc[Create a new project] 
+
+. xref:policies-pdk-create-schema-definition.adoc[Define a schema definition]
++
+Modify the `definition/gcl.yaml` file to define metadata and configuration parameters for your custom policy.
+
+
+. xref:policies-pdk-configure-features.adoc[Configure custom policy features in Rust]
++
+Use the rust code examples to implement your custom logic in the `src/lib.rs` file.
+
+. xref:policies-pdk-compile-policies.adoc[Compile your custom policy]
++
+Compiling your policy creates the `WebAssembly` binary file and configuration files necessary to upload your policy to Exchange and deploy it to a Flex Gateway.
+
+. xref:policies-pdk-debug-local.adoc[Debug Your Policy Locally with PDK]
++
+Use the provided Flex Gateway configuration files to debug your policy locally with PDK before distributing it on Exchange.
+
+
+. xref:policies-pdk-publish-policies.adoc#publish[Publish your policy on Exchange]
+
+.. Publish your policy on Exchange to distribute and test your policy with your Flex Gateway configuration.
+
+.. Test your policy by xref:policies-pdk-deploy-policies.adoc[applying it to an API instance] deployed on Flex Gateway.
```

Comment:
For consistency with other steps, remove the sub-steps and combine into a paragraph: 
Then text your policy...